src/main/resources/default/assets/tycho/scripts/selectableTable.js.pasta

```diff
@@ -32,6 +32,7 @@ sirius.ready(function () {
             _row.addEventListener('click', function (event) {
                 if (event.target.tagName.toLowerCase() !== 'input' &&
                     event.target.tagName.toLowerCase() !== 'a' &&
+                    event.target.tagName.toLowerCase() !== 'i' &&
                     event.target.tagName.toLowerCase() !== 'button') {
```

Comment:
Wouldn't it be better to maintain a whitelist or use a marker class that possible click-targets get assigned?

Reply:
> Icons are often wrapped in `<i>`-tags. If you add an `<a>` which has an `<i>` inside and you manage to click directly on the `<i>` (or the icon) you would select the row. In this case however the intention was to click on the `<a>`-button.

In case of FA, the `<i>` tag only serves as a marker and is replaced by a `<svg>` tag if i am not wrong.